packages/inspire/PerspireServer.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import path from "path";
+import { PerspireView, createPerspireGateway } from "~/inspire";
```

Comment:
The PerspireView can be imported like so:

```import PerspireView from "~/inspire/PerspireView";```

This way the PerspireView forward-export can be removed from inspire/index.js, thus guaranteeing that it won't appear in webpacks nor traversed by tools which import "@valos/inspire".

Reply:
done